src/plugins/intel_gna/src/serial/gna_model_serial.cpp

```diff
@@ -489,6 +512,29 @@ void GNAModelSerial::Export(const GnaAllocations& allocations, std::ostream& os)
         for (auto& tname : output.tensor_names) {
             writeString(tname, os);
         }
+
+        // write pre_processing model
```

Comment:
Please stick to one convention: pre-processing or pre_processing or preprocessing.

Reply:
done